### PR TITLE
(FM-3881) prep for 1.2.1 supported release

### PR DESCRIPTION
**Pull request by ferventcoder**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+##2015-11-24 - Supported Release 1.2.1
+###Summary
+
+Add additional pending reboot scenario. Small bug fix.
+
+###Features
+* Pending reboot - Allow setting a flag directly on provider ([MODULES-2822](https://tickets.puppetlabs.com/browse/MODULES-2822))
```
Comment (Iristyle) [inline CHANGELOG.md]: This affects Windows provider only

Comment (ferventcoder) [inline CHANGELOG.md]: Isn't Windows the only one that has pending scenarios?

Comment (ferventcoder) [inline CHANGELOG.md]: https://github.com/puppetlabs/puppetlabs-reboot#features

Comment (Iristyle) [inline CHANGELOG.md]: Ok, previous `README` didn't call it out, so I'm good with this as-is.  Thanks.

Comment (ferventcoder) [inline CHANGELOG.md]: :+1:

Comment (ferventcoder) [inline CHANGELOG.md]: Anything else holding up a merge?

**Pull request merged**